cli/src/init_command.py

```diff
@@ -0,0 +1,402 @@
+Your Google Cloud project must be enabled for Firebase resources. To do so,
+complete the following steps and then then run the init command again.
+
+1. Enable your Google Cloud project for Firebase resources.
+
+  a. Point your browser to the following URL:
+     {migration_link}
+
+  b. Select your Project ID, which is auto populated in the 'Project name'
+     field, then click 'Continue'.
+
+     If your project name was not auto populated:
+
+       i. Check if your Google Cloud project has already been enabled for
+          Firebase by checking for it here at the URL below, if found skip to
+          step 2:
+
+          {firebase_console_link}
+
+       ii. Verify you have selected the correct Google Cloud project and
+           then run the 'init' command again.
+
+  c. If your Google Cloud project has billing enabled you'll be prompted to
+     confirm the Firebase billing plan, 'Blaze Pay as you go'. Click 'Confirm
+     plan'.
+
+      NOTE: The Debugger will be using the Firebase RTDB service. It's expected
+            the usage will be low enough to fit in the free usage limits
+            specified at {firebase_billing_link}.
+
+  d. Read the 'Few things to remember when adding Firebase to a Google Cloud
+     project' and then click 'Continue'.
+
+  e. (Optional) Enable Google Analytics. For Debugger use this is not required
+     and you can deselect the option.  If however you choose to enable Google
+     Analytics, follow the prompts to select or create a Google Analytics
+     account.
+
+  e. Click 'Add Firebase'.
+
+2. Enable the Firebase Management API if required:
+
+  a. Point your web browser to the following link:
+     {firebase_api_link}
+
+  b. If the Firebase Management API is not reported as 'API Enabled', click
+     'ENABLE' to enable the API. If the Firebase Management API is already
+     enabled, go to step 3.
+
+3. Run the init command again to determine the next required steps.
+"""
+
+DISABLE_FIREBASE_API_MESSAGE = """
+Your project is not yet enabled for Firebase, however the Firebase Management
+API is enabled. Before proceeding you'll need to do the following:
+
+1. Disable the Firebase Management API
+
+  Go to the link below and click 'DISABLE API'
+
+  {firebase_api_link}
+
+2. Rerun the init command.
+"""
+
+CHECK_BILLING_PLAN_MESSAGE = """
+Database '{database_id}' could not be created on project '{project_id}'. One
+potential reason for this failure is that the project is on the Spark billing
+plan. When using the Spark plan, the '--use-default-rtdb' flag must be passed
+with the init command.
+
+Please check your billing plan at {project_billing_plan_link}
+
+If the Spark billing plan is in use, there are two options:
+
+  Option 1. Stay on the Spark plan and have the Snapshot Debugger use the
+            project's default database.
+
+    a. Rerun the init command with the '--use-default-rtdb' flag.
+
+    b. The '--use-default-rtdb' flag must be passed with all CLI commands.
```

Comment:
Potential future improvement:

This and the debuggee id are things that will need to be passed into nearly all CLI commands.  We probably want either environment variables or a config file to store the desired values.